tests/drivers/pwm/pwm_api/boards/stm32l562e_dk.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&timers2 {
+	pwm {
+		st,prescaler = <10000>;
+	};
```

Comment:
according to the 'dts/arm/st: Default "st,prescaler" pwm property to 0' #36133
should the prescaler now be `<0>`

Reply:
> according to the 'dts/arm/st: Default "st,prescaler" pwm property to 0' #36133
> should the prescaler now be `<0>` instead ?

According to [https://github.com/zephyrproject-rtos/zephyr/pull/36133](https://github.com/zephyrproject-rtos/zephyr/pull/36133), added prescaler value <10000> to `stm32l562e_dk.dts` after removing overlay file changes,
the PWM duty cycle forms  `0%... 25%... 50%... 75%...` for pwm api tests on USB logic analyzer.